eng/packaging.targets

```diff
@@ -5,6 +5,7 @@
     <PlaceholderFile>$(MSBuildThisFileDirectory)_._</PlaceholderFile>
     <BeforePack>$(BeforePack);AddNETStandardCompatErrorFileForPackaging</BeforePack>
     <PackageReadmeFile Condition="'$(PackageReadmeFile)' == '' and Exists('PACKAGE.md')">PACKAGE.md</PackageReadmeFile>
+    <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);AddRuntimeSpecificFilesToPackage;IncludeProjectReferencesWithPackAttributeInPackage</TargetsForTfmSpecificContentInPackage>
```

Comment:
You will also need these two properties, otherwise the package layout is incorrect: https://github.com/dotnet/runtime/blob/09608dffab3bf15bf05039b95cea66db42b189aa/eng/packaging.targets#L17-L19